concourse/client/api.py

```diff
@@ -331,3 +379,8 @@ def set_team(self, concourse_team: ConcourseTeam):
 
         team_url = self.routes.team_url(concourse_team.teamname())
         self._put(team_url, json.dumps(body))
+
+
+class ConcourseApiV6_3_0(ConcourseApiV5):
```

Comment:
as just discussed: factor out common methods for v5 and v6 to a third class, and have both versions inherit it, rather than having v6_3 inherit from v5

as an _alternative_, I would also agree with copy-pasting the common methods (since will remove V5 after switching to v6_3 anway)